code/modules/mob/living/simple_animal/animal_defense.dm

```diff
@@ -16,7 +16,23 @@
 		if("grab")
 			grabbedby(M)
 
-		if("harm", "disarm")
+		if("disarm")
+			M.do_attack_animation(src, ATTACK_EFFECT_DISARM)
+			playsound(src, 'sound/weapons/thudswoosh.ogg', 50, TRUE, -1)
+			var/shove_dir = get_dir(M, src)
+			if(!Move(get_step(src, shove_dir), shove_dir))
+				log_combat(M, src, "shoved", "failing to move it")
+				visible_message("<span class='danger'>[M.name] shoves [src]!</span>",
+					"<span class='danger'>You shove [src]!</span>", "<span class='hear'>You hear aggressive shuffling!</span>", COMBAT_MESSAGE_RANGE, src)
+				to_chat(src, "<span class='userdanger'>You're shoved by [M.name]!</span>")
+				return TRUE
+			log_combat(M, src, "shoved", "pushing it")
+			visible_message("<span class='danger'>[M.name] shoves [src], pushing [p_them()]!</span>",
+				"<span class='danger'>You shove [src], pushing [p_them()]!</span>", "<span class='hear'>You hear aggressive shuffling!</span>", COMBAT_MESSAGE_RANGE, src)
+			to_chat(src, "<span class='userdanger'>You're pushed by a shove by [name]!</span>")
```

Comment:
this doesn't roll off the tongue very nicely, how about just this?

```suggestion
			to_chat(src, "<span class='userdanger'>You're pushed by [name]!</span>")
```

Reply:
if you really wanna use the shove verb idk what would work better though

Reply:
The idea is to produce a different message when they actually manage to move you than when they don't.

Reply:
I figured, which is why I used pushed.  The current sentence just sounds unnatural.

Reply:
ah, makes sense